addons/velero/1.2.0/install.sh

```diff
@@ -0,0 +1,91 @@
+
+function velero_pre_init() {
+    if [ -z "$VELERO_NAMESPACE" ]; then
+        VELERO_NAMESPACE=velero
+    fi
+    if [ -z "$VELERO_LOCAL_BUCKET" ]; then
+        VELERO_LOCAL_BUCKET=velero
+    fi
+    if [ -z "VELERO_INSTALL_CLI" ]; then
+        VELERO_INSTALL_CLI="1"
+    fi
+    if [ -z "VELERO_USE_RESTIC" ]; then
+        VELERO_USE_RESTIC="1"
+    fi
+}
+
+function velero() {
+    local src="$DIR/addons/velero/1.2.0"
+    local dst="$DIR/kustomize/velero"
+
+    cp -r "$src/crd" "$dst/"
+    kubectl apply -k "$dst/crd"
+
+    cp "$src/deployment.yaml" \
+        "$src/rbac.yaml" \
+        "$src/service.yaml" \
+        "$dst/"
+
+    if [ "$VELERO_USE_RESTIC" != "0" ]; then
```

Comment:
im not sure this is correct

Reply:
i see the docs now

```
| velero-install-cli | Disable installing the velero CLI with `velero-install-cli=0`. |
| velero-use-restic | Disable the Restic integration with `velero-use-restic=0`. Pod volume data will not be included in backups if Restic is disabled. |
```

typically dont we do something like `disable-...`. its fine either way

Reply:
I'm pretty sure it's correct according to the usage of passing the flag `velero-use-restic=0` but I know that's not an intuitive usage. It's tricky because the default is true and the advanced option in the yaml is `useRestic` so I was trying to avoid changing the flag name.